src/sas/qtgui/Perspectives/Fitting/FittingUtilities.py

```diff
@@ -570,6 +570,31 @@ def plotResiduals(reference_data, current_data):
 
     return residuals
 
+def plotPolydispersities(model, disp_models):
+    plots = []
+    if model is None:
+        return plots
+    # test for model being a sasmodels.sasview_model.SasviewModel?
+    for name in model.dispersion.keys():
+        xarr, yarr = model.get_weights(name)
+        if len(xarr) <= 1: # param name not found or no polydisp.
+            continue
+        # create Data1D as in residualsData1D() and fill x/y members
+        # similar to FittingLogic._create1DPlot() but different data/axes
+        data1d = Data1D(x=xarr, y=yarr)
+        xunit = ""
+        if name in model.details and len(model.details[name]):
+            xunit = model.details[name][0]
+        data1d.xaxis('\\rm{{{}}}'.format(name.replace('_', '\_')), xunit)
+        data1d.yaxis('\\rm{{{weight}}}', 'normalized')
```

Comment:
Too many braces. And could use an r-string.  `'\\rm{{{weight}}}'` => `r'\rm{weight}'`.  I think the latex interpreter just eats them so you won't see it in the output.